.github/scripts/status_update_common.py

```
@@ -0,0 +1,212 @@
+          node(id: $fieldId) {
+            ... on ProjectV2SingleSelectField {
+              options {
+                id
+                name
+              }
+            }
+          }
+        }
+        """
+        variables = {
+            "fieldId": STATUS_FIELD_ID
```

Ruff (0.8.2)

Undefined name `STATUS_FIELD_ID`

(F821)